adminguide.rst

```diff
@@ -200,9 +200,8 @@ flux-core
   prioritization. If building flux-core from source, be sure to configure with
   ``--with-flux-security``. Install on all nodes (required).
 
-flux-sched (optional)
-  The Fluxion graph-based scheduler.  Install on management node
-  (optional, but recommended for production multi-user system installs).
+flux-sched
```

Comment:
I don't have any issues with this change, but for my own curiosity - the commit message says "fluxion ... is required for queues". Is that true? I guess you state that because queues used to partition cannot work with a FCFS scheduler, or are there other features missing to implement queues (at least for testing) in flux-core alone?

Reply:
That might have been too strong of a statement.  Yes, that problem.  I'll reword.